hypervisor/src/hypervisor/linux.rs

```diff
@@ -66,9 +66,21 @@ impl Hypervisor {
                 return Err(anyhow!("Partition \"{}\" already exists", p.name))
                     .lev_typ(SystemError::PartitionConfig, ErrorLevel::ModuleInit);
             }
+
+            let mut veth: Vec<Vec<String>> = Vec::with_capacity(p.veth.len());
+            for p in &p.veth {
+                let v: Vec<String> = p.split(":").map(|s| String::from(s)).collect();
+                if v.len() != 2 || v[0].len() < 1 || v[1].len() < 1 {
+                    return Err(anyhow!("veth pair {p} is malformatted"))
+                        .lev_typ(SystemError::PartitionConfig, ErrorLevel::ModuleInit);
+                }
+
+                veth.push(v);
+            }
```

Comment:
This check should be done at parse-time instead.  
For doing this, consider using a `Vec<(String, String)>` for the `veth` field of the partition configuration with a [custom serde deserializer](https://users.rust-lang.org/t/need-help-with-serde-deserialize-with/18374)  

This may take a lot of time, which is why we probably should make this a separate issue.